app/models/katello/content_view_history.rb

```diff
@@ -66,5 +66,13 @@ def humanized_status
         _("Success")
       end
     end
+
+    def description
+      if self.task.try(:label) == "Actions::Katello::ContentView::Publish"
+        content_view_version.description
```

Comment:
I opened an issue for this: http://projects.theforeman.org/issues/16673. Going to try to take a stab at that and then revisit this.